File: laramie-ingresos-publicos-webapi/src/domain/entities/contribuyente.ts
export default class Contribuyente {

    id: number;
	idPersona: number;
	idTipoPersona: number;
	idTipoDocumento: number;
	numeroDocumento: string;
	nombrePersona: string;
	fechaAlta: Date;

	constructor(
        id: number = 0,
		idPersona: number = 0,
		idTipoPersona: number = 0,
		idTipoDocumento: number = 0,
		numeroDocumento: string = "",
		nombrePersona: string = "",
		fechaAlta: Date = null
	)
	{
        this.id = id;
		this.idPersona = idPersona;
		this.idTipoPersona = idTipoPersona;
		this.idTipoDocumento = idTipoDocumento;
		this.numeroDocumento = numeroDocumento;
		this.nombrePersona = nombrePersona;
		this.fechaAlta = fechaAlta;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.idPersona = row.idPersona ?? 0;
		this.idTipoPersona = row.idTipoPersona ?? 0;
		this.idTipoDocumento = row.idTipoDocumento ?? 0;
		this.numeroDocumento = row.numeroDocumento ?? "";
		this.nombrePersona = row.nombrePersona ?? "";
		this.fechaAlta = row.fechaAlta ?? null;
	}

}

File: modules/ingresos_publicos/models/contribuyente.py
from sqlalchemy import Column, BigInteger, String, DateTime, Boolean
from sqlalchemy.sql import func

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Contribuyente(Base):
    __tablename__ = "ingresos_publicos_contribuyentes"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    id_persona = Column(BigInteger, nullable=False)
    id_tipo_persona = Column(BigInteger, nullable=False)
    id_tipo_documento = Column(BigInteger, nullable=False)
    numero_documento = Column(String(50), nullable=False)
    fecha_alta = Column(DateTime(timezone=True), server_default=func.now(), nullable=False)
    fecha_baja = Column(DateTime(timezone=True), nullable=True)
    activo = Column(Boolean, nullable=False, default=True)
